### PR TITLE
[ID] Add Pass

### PR DESCRIPTION
**Pull request by RaiskyX**
Added translation of "Pass" wiki in Indonesian

### CODE REVIEW

File: wiki/Pass/id.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+tags:
+  - passing
+---
+
+## Pass
```
Comment (wowcake) [inline wiki/Pass/id.md:7]:
```suggestion
# Pass
```

File: wiki/Pass/id.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Sebuah **pass** menjelaskan penyelesaian [beatmap](/wiki/Beatmap) atau bagian beatmap tertentu tanpa menggunakan [mods](/wiki/Game_modifier)[No Fail](/wiki /Game_modifier/No_Fail)) atau masuk ke bagian break dengan lebih dari 50% [health](/wiki/Beatmapping/Health). Dalam kasus mencapai bagian break, indikator akan ditampilkan dengan suara untuk menunjukkan bahwa pemain telah melewati bagian sebelumnya.
```
Comment (wowcake) [inline wiki/Pass/id.md:9]:
```suggestion
Sebuah **pass** adalah istilah yang digunakan untuk menjelaskan ketika pemain menyelesaikan [beatmap](/wiki/Beatmap) atau, bagian beatmap tertentu tanpa menggunakan [mod](/wiki/Game_modifier)(seperti mod [No Fail](/wiki /Game_modifier/No_Fail)) atau saat pemain berhasil masuk ke bagian [Break](/wiki/Beatmap/Break) dengan mempertahankan lebih dari 50% [health](/wiki/Beatmapping/Health). Jika pemain berhasil mencapai bagian break, maka sebuah indikator akan ditampilkan dengan suara untuk memberitahukan bahwa pemain telah berhasil melewati bagian sebelumnya.
```

**Review by wowcake — CHANGES_REQUESTED** [review]

**Review by wowcake — DISMISSED** [review]
should be good for a quick merge

File: wiki/Pass/id.md (hunk 1 above)
Comment (RaiskyX) [reply wiki/Pass/id.md]: ok

File: wiki/Pass/id.md (hunk 2 above)
Comment (RaiskyX) [reply wiki/Pass/id.md]: ok, applied

**Review by wowcake — APPROVED** [review]
rechecked

**Review by TPGPL — APPROVED** [review]

**Pull request merged**